recipes/landsat-util/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.13.1" %}
+
+package:
+  name: landsat_util
+  version: {{ version }}
+
+source:
+  url: https://github.com/developmentseed/landsat-util/archive/v{{ version }}.tar.gz
+  sha256: 68d0f9bcbd4a34512661554f828f8ee391e5ff43ca1a3a4c6176f5f54e0d1313
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+  entry_points:
+    - landsat = landsat.landsat:__main__
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - usgs >=0.1.9,<1.*
+    - requests >=2.7
+    - python-dateutil >=2.5.1,<3.*
+    - numpy >=1.10.4,<2.*
+    - termcolor >=1.1,1.1.*
+    - rasterio >=0.32
+    - six >=1.8
+    - scipy >=0.17
+    - scikit-image >=0.12.3,<1.*
+    - homura >=0.1.3,<1.*
+    - boto >=2.39
+    - polyline >=1.3,1.3.*
+    - geocoder >=1.9
+    - matplotlib >=1.5.1,<2.*
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pytest
+    - mock
+    - jsonschema
+  imports:
+    - landsat
+  commands:
+    - landsat --help
+    - py.test -s -rxs -v -k "not (test_search_json_output or test_search_lat_lon or test_search_pr_correct  or test_search_pr_wrong_input or test_too_many_results or test_search or test_search_with_geojson or test_search_zero_lon or test_geocode)" tests
```

Comment:
It would be nice to have a py.test maker to skip the tests that require the API key.